addons/web/static/src/core/model_field_selector/model_field_selector_popover.js

```diff
@@ -150,7 +150,7 @@ export class ModelFieldSelectorPopover extends Component {
         return this.props.showDebugInput ?? this.props.isDebugMode;
     }
 
-    filter(fieldDefs, path) {
+    async filter(fieldDefs, path) {
```

Comment:
that's weird

Reply:
I commented on the enterprise PR. WIth what I suggest, there's no need to change anything here